tests/functional/firefox/test_accounts.py

```diff
@@ -3,11 +3,23 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 import pytest
+import urllib
 
 from pages.firefox.accounts import FirefoxAccountsPage
 
 
 @pytest.mark.nondestructive
 def test_account_form(base_url, selenium):
-    page = FirefoxAccountsPage(selenium, base_url).open()
-    assert page.is_create_account_form_displayed, "Create account form is displayed for signed out users"
+    page = FirefoxAccountsPage(selenium, base_url, params='').open()
+    page.join_firefox_form.type_email('success@example.com')
+    page.join_firefox_form.click_continue()
+    url = urllib.unquote(selenium.current_url)
+    assert 'email=success@example.com' in url, 'Email address is not in URL'
```

Comment:
I don't know if Selenium is actually clicking with a mouse or with JS. If it's with JS there's a chance the form can be submitted without being visible. Should we also assert the form is displayed?

Reply:
Good question! Selenium fires real click events based on coordinates, rather than simulating mouse events directly on DOM nodes (this is one of the main reasons I favour it over most JS-based solutions). If something isn't visible on a page and you try to click it, a `element not interactable` error will be thrown. I think this is enough to certify that the form is / isn't visible indirectly 👍

Reply:
Then this is good to go!